src/OrchardCore.Modules/OrchardCore.Email/Views/SmtpSettings.Edit.cshtml

```diff
@@ -31,79 +31,97 @@
 
-                <fieldset class="form-group" asp-validation-class-for="Host">
+                <div class="form-group" asp-validation-class-for="Host">
                     <label asp-for="Host">@T["Host name"]</label>
                     <input asp-for="Host" class="form-control" type="text" />
                     <span asp-validation-for="Host"></span>
                     <span class="hint">@T["The SMTP server domain, e.g. <i>smtp.mailprovider.com</i>."]</span>
-                </fieldset>
+                </div>
 
-                <fieldset class="form-group" asp-validation-class-for="Port">
-                    <label asp-for="Port">@T["Port number"]</label>
-                    <input asp-for="Port" class="form-control" type="number" step="any" min="0" max="65535" />
-                    <span asp-validation-for="Port"></span>
-                    <span class="hint">@T["The SMTP server port, usually 25."]</span>
-                </fieldset>
+                <div class="d-flex flex-wrap align-items-center justify-content-stretch">
+                    <div class="form-group flex-fill" asp-validation-class-for="Port">
+                        <label asp-for="Port">@T["Port number"]</label>
+                        <input asp-for="Port" class="form-control" type="number" step="any" min="0" max="65535" />
+                        <span asp-validation-for="Port"></span>
+                        <span class="hint">@T["The SMTP server port, usually 25."]</span>
+                    </div>
 
-                <fieldset class="form-group" asp-validation-class-for="EnableSsl">
+                    <div id="@Html.Id("SelectEncryptionMethod")" class="form-group flex-fill @(Model.AutoSelectEncryption ? "collapse" : "show")" asp-validation-class-for="EncryptionMethod">
+                        <label asp-for="EncryptionMethod">@T["Encryption method to use"]</label>
+                        <select asp-for="EncryptionMethod" class="form-control">
+                            <option value="@SmtpEncryptionMethod.None" selected="@(SmtpEncryptionMethod.None == Model.EncryptionMethod ? true : false)">
+                                @T["None"]
+                            </option>
+                            <option value="@SmtpEncryptionMethod.SSLTLS" selected="@(SmtpEncryptionMethod.SSLTLS == Model.EncryptionMethod ? true : false)">
+                                @T["SSL/TLS"]
+                            </option>
+                            <option value="@SmtpEncryptionMethod.STARTTLS" selected="@(SmtpEncryptionMethod.STARTTLS == Model.EncryptionMethod ? true : false)">
+                                @T["STARTTLS"]
+                            </option>
+                        </select>
+                        <span asp-validation-for="EncryptionMethod"></span>
+                        <span class="hint">@T["The encryption method used when connecting to mail server."]</span>
+                    </div>
+                </div>
+
```

Comment:
Does the checkbox appear below the selection when expanded?

Reply:
this is how it behaves. When inline, and auto select then port fills the row
![image](https://user-images.githubusercontent.com/9623255/57479845-10e41a00-72a7-11e9-99d4-f742c5939148.png)

![image](https://user-images.githubusercontent.com/9623255/57479887-26594400-72a7-11e9-879c-96b85db78c5a.png)